### PR TITLE
feat: include anvil-zksync in foundryup install script

### PR DESCRIPTION
**Pull request by dutterbutter**
Pending release of: https://github.com/matter-labs/era-test-node/pull/444. 

CI expected to fail until era_test_node renamed to `anvil-zksync`

To test:
- checkout this branch
- `cd foundryup-zksync` 
- `./foundryup-zksync`

# What :computer: 
* include anvil-zksync in foundryup install script

# Why :hand:
* allow users to make use of anvil-zksync 

# Evidence :camera:
Include screenshots, screen recordings, or `console` output here demonstrating that your changes work as intended

### CODE REVIEW

**Review by popzxc — DISMISSED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,148 +1,147 @@
...
+
+      - name: Install Rust
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          toolchain: ${{ env.TARGET_RUST_VERSION }}
+
+      - name: Install cargo-nextest
+        uses: taiki-e/install-action@nextest
+
+      - name: Run era-test-node
```
Comment (Karrq) [inline .github/workflows/test.yml:122]:
```suggestion
      - name: Run anvil-zksync
```

File: foundryup-zksync/foundryup-zksync [hunk 2]
```diff
@@ -250,7 +301,7 @@ OPTIONS:
     -p, --path      Build and install a local repository
     -j, --jobs      Number of CPUs to use for building Foundry (default: all CPUs)
     --arch          Install a specific architecture (supports amd64 and arm64)
-    --platform      Install a specific platform (supports win32, linux, and darwin)
```
Comment (Karrq) [inline foundryup-zksync/foundryup-zksync:253]: are we officially dropping support for win32? I think we should document that properly then

File: foundryup-zksync/foundryup-zksync [hunk 3]
```diff
@@ -178,6 +178,59 @@ EOF
       fi
     done
 
+    # Begin anvil-zksync installation
+    say "downloading anvil-zksync"
+
+    # Supported targets for anvil-zksync
+    SUPPORTED_TARGETS=(
```
Comment (Karrq) [inline foundryup-zksync/foundryup-zksync:185]: I understand that the doc change was unintended... but windows doesn't seem supported anyways

Comment (dutterbutter) [reply foundryup-zksync/foundryup-zksync:185]: `anvil-zksync` does not support windows.

**Comment by popzxc** [comment]
@dutterbutter is it time to hit the merge button? :)

**Comment by dutterbutter** [comment]
@popzxc could you review please

**Review by Romsters — APPROVED** [review]

**Review by popzxc — APPROVED** [review]

**Review by nbaztec — APPROVED** [review]

**Pull request merged**